src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RSessionCallback.cs

```diff
@@ -105,7 +105,7 @@ public Task<PlotDeviceProperties> PlotDeviceCreate(Guid deviceId, CancellationTo
         public async Task ViewLibraryAsync(CancellationToken cancellationToken = default(CancellationToken)) {
             await _coreShell.SwitchToMainThreadAsync(cancellationToken);
             var containerFactory = _coreShell.GetService<IRPackageManagerVisualComponentContainerFactory>();
-            _workflow.Packages.GetOrCreateVisualComponent(containerFactory).Container.Show(focus: true, immediate: false);
+            (_workflow.Packages as IRPackageManagerVisual).GetOrCreateVisualComponent(containerFactory).Container.Show(focus: true, immediate: false);
```

Comment:
And later, when we add more features to `IServiceContainer` we can even get something like `_services.GetServiceFromFactory<IPackageManager, IRPackageManagerVisual>(_workflow.Packages)`